plugins/ldap/web_client/views/ConfigView.js

```diff
@@ -0,0 +1,125 @@
+
+var ConfigView = View.extend({
+    events: {
+        'submit .g-ldap-server-form': function (event) {
+            event.preventDefault();
+            this.$('#g-ldap-servers-error-message').empty();
+            this._saveSettings();
+        },
+        'click .g-remove-ldap-server': function (event) {
+            $(event.currentTarget).parents('.panel').remove();
+        },
+        'click .g-ldap-add-server': function () {
+            this.$('.g-ldap-server-accordion').append(newServerTemplate({
+                server: {collapsedClass: 'in'},
+                index: this.$('.g-ldap-server-panel').length
+            }));
+        },
+        'input .g-uri-input': function (event) {
+            const field = $(event.currentTarget);
+            field.parents('.panel').find('.g-ldap-server-title').text(field.val());
+        },
+        'click .g-ldap-test': function (event) {
+            const btn = $(event.currentTarget);
+            const idx = btn.attr('index');
+            const uri = this.$(`#g-ldap-server-${idx}-uri`).val();
+            const bindName = this.$(`#g-ldap-server-${idx}-bindName`).val();
+            const password = this.$(`#g-ldap-server-${idx}-password`).val();
+
+            restRequest({
+                path: 'system/ldap_server/status',
+                type: 'GET',
+                data: {
+                    uri,
+                    bindName,
+                    password
+                }
+            }).done(resp => {
+                btn.girderEnable(true);
+                if (resp.connected) {
+                    this.$(`#g-ldap-server-${idx}-conn-ok`).removeClass('hide');
+                } else {
+                    this.$(`#g-ldap-server-${idx}-conn-fail`).removeClass('hide')
+                        .find('.g-msg').text(resp.error);
+                }
+            });
+
+            btn.girderEnable(false);
+            this.$(`#g-ldap-server-${idx}-conn-ok`).addClass('hide');
+            this.$(`#g-ldap-server-${idx}-conn-fail`).addClass('hide');
+        }
+    },
+
+    initialize: function () {
+        restRequest({
+            type: 'GET',
+            path: 'system/setting',
+            data: {
+                key: 'ldap.servers'
+            }
+        }).done(resp => {
+            this.servers = resp;
+            this.render();
+        });
+
+        this.breadcrumb = new PluginConfigBreadcrumbWidget({
+            pluginName: 'LDAP login',
+            parentView: this
+        });
+    },
+
+    render: function () {
+        this.$el.html(template({
+            servers: this.servers
+        }));
+
+        this.breadcrumb.setElement(this.$('.g-config-breadcrumb-container')).render();
+        sortable.create(this.$('.g-ldap-server-accordion')[0]);
+        return this;
+    },
+
+    _saveSettings: function () {
+        const servers = _.map(this.$('.g-ldap-server-panel'), panel => {
+            const server = {};
```

Comment:
`const` in es6 doesn't refer to internal mutability of the value, it only forbids reassignment of the variable. I have no idea why that would be valuable, maybe some hinting to the GC, but I've been trying to use it where it makes sense since it's been such a heavily documented feature in ES2015.